pennylane/beta/tapes/__init__.py

```diff
@@ -17,4 +17,5 @@
 """
 from .circuit_graph import NewCircuitGraph
 from .tape import QuantumTape
+from .qubit_param_shift import QubitParamShiftTape
```

Comment:
Wonder why this is showing up :thinking:

Reply:
@ixfoduap I believe the extra files are showing up due to the merge-in of `master`. It should be ok, but one option is to revert that commit and then do `git merge mixed_simulator2`